architecture/paymasters.mdx

```diff
@@ -14,6 +13,25 @@ The implementation of Paymasters significantly improves the user experience by o
 - **NFT-Based Fee Waivers:** An NFT can be created that allows holders to bypass paying transaction fees, adding value to the NFT while providing users with benefits.
 - **Alternative Gas Payments:** Users can pay gas fees with an ERC-20 token other than SOPH, allowing for value accrual and expanding the range of token options for gas payment.
 
+## How to create your own paymaster
+
+You can create your own paymaster with your custom logic to subsidize users in multiple ways.
+
+### Deploy via txSync
+
+[txSync Tsuko](https://app.txsync.io/tsuko) offers everyone an easy way to create simple sponsored paymasters on Sophon and Sophon testnet. To create one, please follow our [walkthrough video](https://www.loom.com/share/bf5d2d2d774646eaa2aed3c71a822263?sid=7ba4d65f-465b-442e-b22a-30f52d75997d). 
+
+Via **txSync Tsuko**, you can do various things with your paymasters:
```

Comment:
Given the walkthrough doesn't provide further detalis on restrictions, I think we should give a few examples on types of restrictions devs should add to it (after all you don't want an open paymaster), and also link to some tsuko  docs about it